lib/matplotlib/tests/baseline_images/test_image/image_shift.svg

```diff
@@ -26,9 +26,9 @@ L 72 43.2
+   <g clip-path="url(#pa943344acd)">
+    <image height="99" id="image784d7f5ce5" transform="matrix(4.525253 0 0 3.505051 72 41.8)" width="99" xlink:href="data:image/png;base64,
```

Comment:
another image size change, but now I am guessing that this is all accounted for by the subsequent transform.